### DISCUSSION TITLE
请问有使用 VSCode LaTeX Workshop 插件以 XeLaTeX 方式编译最新版硕士论文的方式吗

### DISCUSSION THREAD

**Discussion by mrdrivingduck** [nuaatug/nuaathesis#67] [Q&A] [answered]
在网上试了几个 enable XeLaTeX 的法子，好像还是会报错捏

**Comment by yzwduck** [comment, marked as answer]
VSCode 里的 LaTeX Workshop 插件支持 latexmk (latexmkrc) 编译，我测试下来，应该是开箱即用的体验：
1. VSCode 打开论文目录；
2. 激活 Workshop 插件；
3. Build LaTeX project > Recipe: latexmk (latexmkrc).

如果无法顺利用上 latexmk，请检查下列内容：
* latexmk 是否有安装？在命令行里运行 `latexmk` 应该会输出它的版本号；如果没有的话，用 TeXLive 或者 MiKTex 安装上；
* 主 latex 文件 (master.tex) 所在目录下，是否有 latexmkrc 文件？它是 latexmk 的配置文件；
* VSCode 打开的目录是否为 demo_chs (或者你的论文目录)？不推荐打开 nuaathesis 目录，因为它是模板开发的目录，而不是论文的目录。
